server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java

```diff
@@ -2298,6 +2354,7 @@ IndexMetadata build(boolean repair) {
                 isSystem,
                 INDEX_HIDDEN_SETTING.get(settings),
                 timestampRange,
+                eventIngestedRange,
```

Comment:
Can we assert that it is `!= null` in the `build` method?

Reply:
Added in next push.